build.sh

```diff
@@ -1,26 +1,23 @@
 #!/usr/bin/env bash
+# arg1: name of destination dockerhub
+# arg2: dockerhub username
+# arg3: dockerhub password
+
 set -x -e
 
 buildnumber=${4-$(date -u +"%y%m%d%H%M")}
 
-# docker build --no-cache -t "$1"/php:5.6.21-apache_"$buildnumber" 5.6.21-apache
-# docker build --no-cache -t "$1"/php:5.6.21-apache-xdebug_"$buildnumber" 5.6.21-apache-xdebug
-# docker build --no-cache -t "$1"/php:7.0.6-apache_"$buildnumber" 7.0.6-apache
-# docker build --no-cache -t "$1"/php:7.0.6-apache-xdebug_"$buildnumber" 7.0.6-apache-xdebug
-# docker build --no-cache -t "$1"/php:7.2.1-apache-xdebug_"$buildnumber" -t "$1"/php:latest-xdebug_"$buildnumber" 7.2.1-apache-xdebug
-docker build --no-cache -t "$1"/php:7.2.1-apache_"$buildnumber" -t "$1"/php:latest_"$buildnumber" 7.2.1-apache
+docker build -q --no-cache -t "$1"/php:5.6.36-apache_"$buildnumber" 5.6.36-apache
+docker build -q --no-cache -t "$1"/php:7.0.30-apache_"$buildnumber" 7.0.30-apache
```

Comment:
we dont build the xdebug version?

Reply:
I moved those to build-xdebug.sh

Reply:
the xdebug images need the new base images to be in appsvc and needs to know the new buildnumber, building both at the same time wont work.